sample-files/MyElementEventListener.java

```diff
@@ -0,0 +1,117 @@
+package com.paypal.selion.testcomponents;
+
+import com.paypal.selion.configuration.Config;
+import com.paypal.selion.configuration.Config.ConfigProperty;
+import com.paypal.selion.platform.grid.Grid;
+import com.paypal.selion.platform.html.Button;
+import com.paypal.selion.platform.html.CheckBox;
+import com.paypal.selion.platform.html.DatePicker;
+import com.paypal.selion.platform.html.Form;
+import com.paypal.selion.platform.html.Image;
+import com.paypal.selion.platform.html.Label;
+import com.paypal.selion.platform.html.Link;
+import com.paypal.selion.platform.html.RadioButton;
+import com.paypal.selion.platform.html.SelectList;
+import com.paypal.selion.platform.html.Table;
+import com.paypal.selion.platform.html.TextField;
+import com.paypal.selion.platform.html.support.events.AbstractElementEventListener;
+import com.paypal.selion.platform.html.support.events.Checkable;
+import com.paypal.selion.platform.html.support.events.Clickable;
+import com.paypal.selion.platform.html.support.events.Deselectable;
+import com.paypal.selion.platform.html.support.events.Selectable;
+import com.paypal.selion.platform.html.support.events.Submitable;
+import com.paypal.selion.platform.html.support.events.Typeable;
+import com.paypal.selion.platform.html.support.events.Uncheckable;
+import com.paypal.selion.reports.runtime.WebReporter;
+
+public class DebugScreenshotListener extends AbstractElementEventListener {
```

Comment:
@renelux On the `gh-pages` branch we try to keep samples to 2 spaces per indent and 80 columns wide. This helps (a bit) with browsers/screen that do not have a wide viewport. Sorry for the deviation from the standard on the `develop` branch. It's the a low-tech solution, i know. Will you please update this sample source